modules/enums.py

```diff
@@ -25,3 +25,11 @@ class ViewEstablishmentEnums(IntEnum):
 
     FOLLOW = 8
     REMAIN = 9
+
+
+class ReplicationEnums(Enum):
```

Comment:
Do we need these to be `IntEnum`s later?

Reply:
Yes, we will send them in our msgs. Changed to `IntEnum`